particles/WasmTemplate/example.arcs

```diff
@@ -0,0 +1,27 @@
+schema Product
```

Comment:
Since it's an example, header docs explaining what this file is would likely help a new reader.

Reply:
I didn't want to explain how Arcs works in general. If someone's reading this file as a primer on how to use Arcs or write particles in general, we should steer them to the real docs.

Note also I do intend to write much more complete documentation once I've got some basic unit testing in place.